PROJECT

```diff
@@ -79,4 +79,36 @@ resources:
   kind: Standalone
   path: github.com/splunk/splunk-operator/api/v3
   version: v3
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: splunk.com
+  group: enterprise
+  kind: IndexerCluster
+  path: github.com/splunk/splunk-operator/api/v4
+  version: v4
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: splunk.com
+  group: enterprise
+  kind: Standalone
+  path: github.com/splunk/splunk-operator/api/v4
+  version: v4
+- api:
+    crdVersion: v1
```

Comment:
i couldnt find ClusterManager and LicenseManager v4 . is this missing?

Reply:
Indeed it was missing. I rerun `operator-sdk create api ` for LM & CM to update the PROJECT, then restored the files `api/v4`.